### ISSUE TITLE
dvanalysis GUI - table view = buggy

### ISSUE THREAD

**Issue by libbyhopfauf** [mipops/dvrescue#538]
Within the table view of the analysis GUI, when you click on a line in the table, the following things happen as expected:

- the line in the table is highlighted in purple
- the graph and the player respond by going to that frame

However, if you press play in the player or click to another line in the table, the highlighting disappears. It will scroll along the table for a little bit (highlighting each row as the player goes past that specific frame), but then the highlighting disappears. It would be great if it stayed on the last "interesting frame" until it gets to another one and scrolls in time with the player content and the plots on the graph. 

Also, the highlighting on the table and corresponding jump to the same frame on the graph and the player seems to be inconsistent. You can hear a quick sound, but the rest of the functions seems to either lag or not respond at all. [Here](https://drive.google.com/file/d/11mrpw56flWDFZFCeh24cR1yAD5t3WpQg/view?usp=sharing) is a screen recording I did to demonstrate what I'm talking about (I added a "warp" effect so that you could tell when I was clicking vs. when the program was responsive).

**Comment by libbyhopfauf** [comment]
Using dvrescue version 20220820 the dvanalysis tab is still really buggy. Similarly to issues I've experienced with dvcapture in the GUI, the table interaction seems to start out fine for the first 4-5 selections made in the table, then becomes slightly inconsistent (notice in the video below how on the 6th click, there is no response and then a severe lag for the 7th) and after about 10 clicks becomes really buggy, unresponsive at times and unpredictable. Once it becomes buggy, every ~10th time I click on a line in the table, it jumps all the way back to the beginning, but the display in the player and graph do not necessarily change to the beginning (in the video below, this starts to happen after the 10th click and happens multiple times across the 4 minute video).

Also, the segmentation still not responding even though there are clearly changes in the audio and the recording time (as displayed/noted in the table), but segmentation doesn't occur even when the applicable segmenting rules are selected. For more details, please see #537.

Here is another screen recording to demonstrate the quick decline to buggy (again, I added a "warp" effect so that you could tell when and where I was clicking vs. when the program was responsive):


https://user-images.githubusercontent.com/13877631/186003583-5d3cb503-9728-4b00-bed5-a3ea4683dedf.mp4

**Comment by ElderOrb** [comment]
@libbyhopfauf can you please check https://github.com/mipops/dvrescue/pull/551 ?

**Comment by libbyhopfauf** [comment]
@ElderOrb when I checked this version, nothing is generated in the table or graph view in the dvanalyser tab. The video will play, but even after letting the file load for 30 minutes, nothing appears in the table or the graph. It also seems like I can't click through to a specific point in the video. 

<img width="1792" alt="Screen Shot 2022-09-12 at 11 49 04 AM" src="https://user-images.githubusercontent.com/13877631/189732949-259605e7-ed73-416d-80d0-31bc1459e3a8.png">

**Comment by ElderOrb** [comment]
@libbyhopfauf  What version of Qt have you used? Also, can you share video so I can try on my side?

**Comment by libbyhopfauf** [comment]
@ElderOrb I've tried versions 6.3.1_2 and 6.3.1_3 for qt on a 2012 Mac Mini (running Catalina 10.15.7) and 6.3.1_3 for qt on a 2019 Macbook Pro (running Big Sur 11.6.7) and I've been able to get some XML and DV files to load, but for anything with a lot of errors, the DVRescue GUI crashes. And the segmenting rules are still not working for any files in both the analysis and packager tabs.  

Here's the [DV file](https://drive.google.com/file/d/1jEjq-hLCurRntW6eKtvQZQOny3ye4Pqa/view?usp=sharing) and corresponding [XML](https://drive.google.com/file/d/11VvxyTvM4S3GqlyBNmjoHUmQQxhTmDql/view
) that I was trying to use.

In #537 I also describe trying to previously package and analysis this same file and the segmenting rules don't seem to work in analysis or packager.
